config/routes.rb

```diff
@@ -1,10 +1,12 @@
 Rails.application.routes.draw do
-  get 'sessions/new'
-  resources :tasks
-  root to: "tasks#index"
-  resources :users, only: [:new, :create, :show, :edit, :update]
-  resources :sessions, only: [:new, :create, :destroy]
+  resources :users, only: [:new, :create, :show, :update]
```

Comment:
`%i`記法を使うとスペース区切りで簡潔に記載できます。
```suggestion
  resources :users, only: %i[new create show update]
```